demos/code/code.js

```diff
@@ -158,8 +158,10 @@ Code.loadBlocks = function(defaultXml) {
  */
 Code.changeLanguage = function() {
   // Store the blocks for the duration of the reload.
+  // This should be skipped for the index page, which has no blocks and does
+  // not load Blockly.
   // MSIE 11 does not support sessionStorage on file:// URLs.
-  if (window.sessionStorage) {
+  if (typeof Blockly != 'undefined' && window.sessionStorage) {
```

Comment:
Revert this line. `Blockly` will never equal the string `'undefined'`.

Reply:
I also have no knowledge of this change.